File: soutLink-backend/src/main/java/com/soutLink/student/dto/SupervisorAppointmentResponse.java
package com.soutLink.student.dto;

import com.soutLink.enums.AppointmentStatus;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class SupervisorAppointmentResponse {
    private Long id;
    private String subject;
    private String supervisorName;
    private AppointmentStatus status;
    private String date;
    private String time;
}

File: soutLink-frontend/src/app/services/models/supervisor-appointment-response.ts
/* tslint:disable */
/* eslint-disable */
export interface SupervisorAppointmentResponse {
  date?: string;
  id?: number;
  status?: 'WAITING' | 'PENDING' | 'ACCEPTED' | 'CANCELED';
  subject?: string;
  supervisorName?: string;
  time?: string;
}
